skyportal/handlers/api/candidate.py

```diff
@@ -289,6 +290,9 @@ def get(self, obj_id=None):
                 )
             ]
             candidate_list.append(obj.to_dict())
+            candidate_list[-1]["comments"] = obj.get_comments_owned_by(
+                self.current_user
+            )
```

Comment:
Could you please add sorting in a reversed order here? Currently, I see:
![image](https://user-images.githubusercontent.com/7557205/92528382-8898c080-f1dd-11ea-8c5d-60016bccf06f.png)